articles/postgresql/concepts-connection-libraries.md

```diff
@@ -11,10 +11,10 @@ ms.topic: article
 ms.date: 09/26/2017
 ---
 # Connection libraries for Azure Database for PostgreSQL
-This topic lists libraries and drivers that developers can use to program applications to connect to and query Azure Database for PostgreSQL.
+This topic lists libraries and drivers that developers can use to develop applications to connect to and interact with Azure Database for PostgreSQL.
 
 ## Client interfaces
-Most language client libraries used to connect to PostgreSQL server are external projects and are distributed independently. These are supported on the Windows, Linux, and Mac platforms. Some of the popular client drivers are listed in the following table:
```

Comment:
Not sure if "supported" is the correct term. It implies that someone will help resolve issues when instead I think the documentation wants to say that they work in those platforms.

Reply:
Agreed. **Supported** is an overloaded term. Can mean someone supports them, bug get fixed, Microsoft has a support team for it, etc. I think it means "work on" in this context.

Reply:
The intent here was, these don't only work on multiple platforms but they are supported by authors/azure on these platforms.